### DISCUSSION TITLE
Troubleshooting VTherm over climate not following TRV device setpoint

### DISCUSSION THREAD

**Discussion by AliveDevil** [jmcollin78/versatile_thermostat#733] [Q&A] [answered]
Sparked by a comment in [#690](https://redirect.github.com/jmcollin78/versatile_thermostat/issues/690#issuecomment-2558379422) mentioning last seen.
Is the follow underlying target temperature directly tied to the last seen attribute?
As the description of that field "Last seen room temperature sensor entity id. Should be datetime sensor" doesn't indicate this is linked to the target temperature, but the current temperature as measured from any external sensor.

Should follow underlying target temperature work in case last seen isn't configured?

**Comment by AliveDevil** [comment]
Found an issue here: https://github.com/jmcollin78/versatile_thermostat/blob/53160f1365225582acf23d0076fbfa61ab2cfdac/custom_components/versatile_thermostat/thermostat_climate.py#L712-L713
My TRV allows a setpoint of 0.5 degree steps.
When I change from 18 to 18.5, or similar, the event is dropped, for a too-small change.
This should really be `-(VTherm Step) < diff < (VTherm Step)` - in my case, I set these to 0.5 instead of 0.1, because I don't need 0.1 precision.

E: Changed to permalink.

**Reply by jmcollin78** [reply]
Oooh yes. You are right. (and that is ugly)

**Reply by jmcollin78** [reply]
https://github.com/jmcollin78/versatile_thermostat/pull/736

**Comment by jmcollin78** [comment]
> Is the follow underlying target temperature directly tied to the last seen attribute?

No. There is no link.

**Comment by jmcollin78** [comment, marked as answer]
https://github.com/jmcollin78/versatile_thermostat/releases/tag/6.8.4
